### PR TITLE
Article on how to use notebooks plus minor doc edits.

### PR DESCRIPTION
**Pull request by heistermann**
Addresses issues #91 and #83 (to a minor extent). 

I think this article is important to make sure users understand and master the transition to notebooks.

Also added a rough layout and some content of a Python intro (issue #92). This intro should guide users to Python issues that we consider as important. This should be further developed in the future.

### CODE REVIEW

File: doc/source/jupyter.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+
+Interactive examples with jupyter notebooks
+-------------------------------------------
+
+All :math:`\omega radlib` examples are distributed as `jupyter notebooks <http://jupyter.org/>`_. This way, you can interactively explore various  topics. Each notebook is organized in documented code blocks. You can browse through a notebook block by block, inspect the results, and experiment with the code. However, you can also view the rendered notebooks including the example results on the web pages of this section: Each page was directly generated from :math:`\omega radlib`'s example notebooks. This way, you can copy&paste code snippets directly into your applications.
```
Comment (kmuehlbauer) [inline doc/source/jupyter.rst]: two white spaces between "various" and "topics"

File: doc/source/jupyter.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+The notebooks are distributed with each :math:`\omega radlib` release on `PyPI <https://pypi.python.org/pypi/wradlib>`_ (beginning with version 0.9.0). Alternatively, you can `download <https://github.com/wradlib/wradlib/archive/master.zip>`_ the latest ("bleeding edge") notebooks from our `GitHub repository <https://github.com/wradlib/wradlib>`_. In both cases, you find everything in the directory ``/notebooks`` (after extracting the downloaded archives).
+
+
+.. _ref_get_data:
+
+How can I get the example data?
+-------------------------------
+
+Most notebooks use example data. These data are provided in a seperate repository and you need to "install" them manually. *"Installing"* actually means: Download the data archive `here <https://github.com/wradlib/wradlib-data/archive/master.zip>`_ and extract it into any arbitrary directory. Now you need to set an environment variable pointing to that directory:
```
Comment (kmuehlbauer) [inline doc/source/jupyter.rst]: should read: separate

File: doc/source/jupyter.rst [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+After this procedure, the example notebooks will automagically pull the required files from the data archive.
+	
+
+.. _ref_get_jupyter:
+	
+How to install jupyter?
+-----------------------
+
+As already pointed out above, you can just look at the rendered notebooks :doc:`online docs <notebooks>`. In order to use them interactively, you need to install ``jupyter``. ``jupyter`` is shipped with with `Anaconda's <https://www.continuum.io/downloads>` distribution by default. If you installed wradlib into Anaconda's *default (root) environment*, you are already done. If you installed wradlib in a seperate *virtual environment* (as recommended :doc:`here <gettingstarted>`), you need to install ``jupyter`` in that virtual environment, too::
```
Comment (kmuehlbauer) [inline doc/source/jupyter.rst]: double "with"

**Comment by kmuehlbauer** [comment]
I just had a quick glance, but this is LGTM. I restarted the failed py3.4 build, seems it was an temporary issue with fetching dependencies. Seems to work properly now. 

@heistermann Merge at your discretion. I only found some typos and stuff.

**Comment by heistermann** [comment]
I'd like to include a rough layout for the Python intro (issue #92), although this does not have to be completed for the release in my opinion.

**Comment by heistermann** [comment]
Seems good enough for me to merge. Python intro need to be further developed in the near future.

**Comment by kmuehlbauer** [comment]
Cool!

**Pull request merged**